source/trading/Mapping_symbols.rst

```diff
@@ -2,42 +2,79 @@
 .. _`/instruments`: https://www.tradingview.com/rest-api-spec/#operation/getInstruments
 .. _`/mapping`: https://www.tradingview.com/rest-api-spec/#operation/getMapping
 
-Mapping symbols
+Symbol mapping
 ---------------
 
-What is mapping
-...............
-We call *mapping symbols* the matching between the names of the broker's instruments and TradingView.
-This mapping solves the problem of TradingView and broker symbol names mismatching. If the broker's doesn't use 
-TradingView\'s data, then mapping is necessary.
+*Symbol mapping* is the matching between the broker's and TradingView symbols.
+It helps both brokers and TradingView operate with symbols in a convenient way and avoid symbol mismatching.
 
-How to implement mapping
-........................
-Mapping is set with the `/mapping`_ endpoint implementation. This endpoint must be accessible without 
-authorization. In TradingView production, it is automatically requested once a day. Based on the response to the 
-request, a mapping of instruments is generated on the TradingView side. In TradingView staging, the `/mapping`_ 
-request is made manually if necessary. At the development stage, you can set a partial mapping, i.e. not for all 
-instruments supported by the broker.
+When to implement
+..................
+
+The mapping is required when you use TradingView data that is available from a third-party source.
+If you plan to use *only* your symbols, you don't need to implement mapping.
+However, if you decide to use both TradingView (available from a third-party source) and your own data, you must implement mapping for all symbols, including yours.
+
+How to implement
+.................
+
+To implement symbol mapping, use the `/mapping`_ endpoint.
+
+.. note::
+  `/mapping`_ must be accessible without authorization.
+
+The endpoint must return an object in the following format:
+
+.. code-block:: json
+
+  {
+    "symbols": [
+      {
+        "f": ["EURUSD"],
+        "s": "FX_IDC:EURUSD"
+      },
+      {
+        "f": ["AAPLE"],
+        "s": "NASDAQ:AAPLE"
+      }
+    ],
+    "fields": ["brokerSymbol"]
+  }
+
+Here, ``symbols`` is an array of objects describing symbols. Every object contains two required properties:
```

Comment:
`symbols` is a self-explanatory name, so I would suggest to remove the description. 
```suggestion
Here, ``symbols`` is an array of objects that contain two required properties:
```